src/components/special/_win95.less

```diff
@@ -0,0 +1,30 @@
+@import url("https://fonts.googleapis.com/css2?family=Questrial&display=swap");
+
+.win95-btn {
+  font-family: "Questrial", sans-serif;
+  font-size: inherit;
+  color: black;
+  background-color: #d4d4d4;
+  padding: 0.3rem 3.4rem;
+  border-bottom: 3px solid black;
+  border-right: 3px solid black;
+  border-left: 3px solid white;
+  border-top: 3px solid white;
+  margin-right: 2.6rem;
+  box-shadow: 0 0 0 black;
+  transition: all 0.2s;
+}
+
+.win95-btn:last-child {
+  margin: 0;
+}
+
+.win95-btn:hover {
+  box-shadow: 0.4rem 0.4rem 0 black;
```

Comment:
Reduce translate and box-shadow on hover I think it is a little too much

Reply:
thanks for the feedback! I made the changes.